.github/workflows/build-pr.yml

```
@@ -147,4 +147,49 @@ jobs:
         env:
           XATA_DATABASE_URL: ${{ secrets.INTEGRATION_TEST_DATABASE_URL }}
           XATA_API_KEY: ${{ secrets.INTEGRATION_TEST_API_KEY }}
-          XATA_DATABASE_BRANCH: ${{ secrets.INTEGRATION_TEST_BRANCH }}
+          XATA_BRANCH: ${{ secrets.INTEGRATION_TEST_BRANCH }}
+
+  e2e-test:
+    name: E2E Test
+    runs-on: ubuntu-latest
+    needs: build
+    #if: github.ref_name == 'changeset-release/main'
```

Enable condition before merging

```suggestion
    if: github.ref_name == 'changeset-release/main'
```